docs/guide/rsun.rst

```diff
@@ -0,0 +1,62 @@
+.. _topic-guide-rsun:
+
+*********************************
+The role of ``rsun`` in ``sunpy``
+*********************************
+
+When a ``map.wcs`` is constructed, a key variable is ``rsun``; the height at which emission originates.
+Data providers often include this information for end users via the fits keyword: ``RSUN_REF``,
+where this keyword is not available ``sunpy`` assumes a standard value defined by `sunpy.sun.constants`.
+
+In theory, the radiation detected by a single filter, especially for coronal emission,
+comes from a variety of heights and ``rsun`` is only a loose approximation.
+In practice though, the value of ``rsun`` is important when working with coordinates and transforms, it provides 3 dimensional information about the location of a map pixel.
+Idiosyncrasies can occur which can be pre-empted or fixed by an understanding of behaviors surrounding ``rsun``.
+
+Reprojecting between frames with different ``rsun``
+===================================================
+
+When the values of ``rsun`` from two wcs instances are different, issues with reprojecting between those frames can be encountered:
+:meth:`~sunpy.map.GenericMap.reproject_to` by default enforces a round-trip behavior,
+the idea being that you should only trust the reprojection if the 2D coordinates from each observer both resolve to the same 3D point (within a pixel volume).
+When the values for ``rsun`` are different, that criterion fails towards the limb.
+In other cases, this furthers results in banding as the criterion fails, then succeeds and then fails again.
+
+Changing the value of ``rsun`` can fix this issue:
+Take the example, :ref:`sphx_glr_generated_gallery_map_transformations_reprojection_different_observers.py`
+If we run the reprojection twice, once before fixing the discrepancy in the ``rsun_ref`` metadata and once after:
+
+.. code-block:: python
+
+    >>> import sunpy.map
+    >>> from sunpy.map import Map
+    >>> from sunpy.data.sample import AIA_193_JUN2012, STEREO_A_195_JUN2012
+    >>> map_aia = Map(AIA_193_JUN2012)
+    >>> map_euvi = Map(STEREO_A_195_JUN2012)
+    >>> outmap1 = map_euvi.reproject_to(map_aia.wcs)
+    >>> map_euvi.meta['rsun_ref'] = map_aia.meta['rsun_ref']
+    >>> outmap2 = map_euvi.reproject_to(map_aia.wcs)
+
+we can see the difference in the appearance of the reprojected maps at the limb.
+
+Transforming between ``helioprojective`` frames
```

Comment:
Would it make more sense to have this first, as the reproject_to example is technically a special case of this.